specs/spec_reader_test.go

```diff
@@ -1,15 +1,37 @@
 package specs
 
-// func TestLoadSpecs(t *testing.T) {
-// 	specReader, err := NewSpecReader("testdata")
-// 	if err != nil {
-// 		t.Fatal(err)
-// 	}
-// 	for fileName, spec := range specReader.sources {
-// 		t.Run(fileName, func(t *testing.T) {
-// 			if !reflect.DeepEqual(spec, testSpecs[fileName]) {
-// 				t.Errorf("expected spec %s to be:\n%v\nbut got:\n%v", fileName, testSpecs[fileName].Spec, spec.Spec)
-// 			}
-// 		})
-// 	}
-// }
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+var sources = map[string]Source{
+	"aws.cq.yml": {
+		Name:          "aws",
+		Path:          "aws",
+		Version:       "v1.0.0",
+		MaxGoRoutines: 10,
+		Registry:      RegistryLocal,
+	},
+}
+
+var destinations = map[string]Destination{
+	"pg.cq.yml": {
```

Comment:
I think we've dropped the convention of having `.cq` as part of the of config file names now, let's remove it from the tests too so as to not confuse anyone :)

Reply:
Thanks! Done in https://github.com/cloudquery/plugin-sdk/pull/110/commits/7bc23382b977fad5b0ec34a525d5a595289f36c5